legal-api/src/legal_api/services/filings/validations/continuation_in.py

```diff
@@ -171,4 +173,7 @@ def validate_business_in_colin(filing_json: dict, filing_type: str) -> list:
         elif legal_name != response_json['business']['legalName']:
             msg.append({'error': 'Legal name does not match with company legal name from Colin.',
                         'path': business_legal_name_path})
+        elif founding_date != response_json['business']['foundingDate']:
+            msg.append({'error': 'Founding date does not match with founding date from Colin.',
```

Comment:
Does this code handle the case where COLIN did not provide the date but the user entered it in the UI/filing?

Should we let the user override the date from COLIN, in case it was wrong?

Reply:
there are 2 dates

1. incorporation date in foreign jurisdiction, which user can override `filing_json['filing'][filing_type]['foreignJurisdiction']['incorporationDate']`
2. foundingDate of expro in BC, which user cannot override `filing_json['filing'][filing_type]['business']['foundingDate']`

Reply:
The date of incorporation entered in the filing UI will not change the founding date:
![image](https://github.com/user-attachments/assets/6828b31f-b670-4ec0-8a46-0017f9416bc1)

It changes the foreignJurisdiction.incorporationDate
![image](https://github.com/user-attachments/assets/fa7dbc4c-b534-4fa9-8d4d-560dc136c80d)